### ISSUE TITLE
508-defect-2 [USABILITY, MOBILE DESIGN]: Interactive elements SHOULD have at least 24px space between

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12671]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

There are two links that are close together, as shown in the screenshot, on the 4 of 5 Additional Information screen. It is an interaction design best practice, and a WCAG 2.2 draft success criteria, that there **should** be generally at least 1.5 rem (around 20 or 30 pixels) spacing between interactive elements. 

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a touch device user, I want to select a link and not accidentally hit another one because it is too close.

## Environment

* Operating System: all
* Browser: all
* Server destination: staging

## Steps to Recreate

1. Logged in as 261, enter https://staging.va.gov/disability/file-disability-claim-form-21-526ez/fully-developed-claim in browser
2. Navigate to 4 of 5 Additional Information
3. Verify the links appear as shown in the screenshot below

## Possible Fixes (optional)

To provide more spacing between the links, recommend making the two items into a list, and ensuring there is 1.5 rem (24px) between the links.

### Current code
```diff
<div>
  <h3 class="vads-u-font-size--h5">
    Fully developed claim program
  </h3>
  <p>You can apply using the Fully Developed Claim (FDC) program if you’ve uploaded all the supporting documents or additional forms needed to support your claim.</p>

! <a href="/disability/how-to-file-claim/evidence-needed/fully-developed-claims/" target="_blank" rel="noopener noreferrer">Learn more about the FDC program</a>.

! <br><a href="/disability/how-to-file-claim/evidence-needed/" target="_blank" rel="noopener noreferrer">View the evidence requirements for disability claims</a>.

</div>
```

### Suggested code
```diff
<div>
  <h3 class="vads-u-font-size--h5">
    Fully developed claim program
  </h3>
  <p>You can apply using the Fully Developed Claim (FDC) program if you’ve uploaded all the supporting documents or additional forms needed to support your claim.</p>

+  <ul>
+    <li><a href="/disability/how-to-file-claim/evidence-needed/fully-developed-claims/" target="_blank" rel="noopener noreferrer">Learn more about the FDC program</a>.</li>

+    <li><a href="/disability/how-to-file-claim/evidence-needed/" target="_blank" rel="noopener noreferrer">View the evidence requirements for disability claims</a>.</li>
+  </ul>

</div>
```

## WCAG or Vendor Guidance (optional)

* [W3C Understanding WCAG **2.2** - Success Criterion 2.5.8 Pointer Target Spacing (Level AA)](https://w3c.github.io/wcag/understanding/pointer-target-spacing)

* [Make clickable elements large, and put space between them. ](http://web-accessibility.carnegiemuseums.org/content/buttons/)

## Screenshots or Trace Logs

![Screen Shot 2020-08-20 at 1 33 49 PM](https://user-images.githubusercontent.com/57469/90805731-073ec400-e2ea-11ea-979b-f516bcaa2b71.png)

**Comment by Mottie** [comment]
Hi @jenstrickland!

I have a question about wrapping the links in a `<p>` tag versus adding them to a list as you suggested.

When I add the links to a list,

<details><summary>the bullets are visible</summary>


![Screen Shot 2020-09-16 at 8 59 31 AM](https://user-images.githubusercontent.com/136959/93347841-27837500-f7fb-11ea-815d-87d35ccfde50.png)</details>

I don't think hiding the bullets is a good practice, but I'm not sure?

What if we wrapped each link inside a paragraph tag?

<details><summary>Links in paragraphs</summary>


![Screen Shot 2020-09-16 at 9 13 36 AM](https://user-images.githubusercontent.com/136959/93349344-f6a43f80-f7fc-11ea-8650-59235e274cf1.png)</details>

**Comment by jenstrickland** [comment]
@Mottie My 2 cents is having the bullets visible clarifies, and using a list improves the experience for those with cognitive considerations by styling each distinctly, as well as calling out each separately for screen readers. Without the list semantics, they are strung together links.

**Comment by Mottie** [comment]
Ok, sounds good. Thanks!

**Comment by Mottie** [comment]
This fix has been merged into master and will be available in staging shortly.

**Comment by jenstrickland** [comment]
Validated. 🎉
